scripts/just-task.js

```diff
@@ -21,65 +23,72 @@ const perfTest = require('./tasks/perf-test');
 
 let packageJson;
 
-option('production');
-
-// Adds an alias for 'npm-install-mode' for backwards compatibility
-option('min', { alias: 'npm-install-mode' });
-
-option('webpackConfig', { alias: 'w' });
-
-// Build only commonjs (not other TS variants) but still run other tasks
-option('commonjs');
-
-registerTask('clean', clean);
-registerTask('copy', copy);
-registerTask('jest', jest);
-registerTask('jest-watch', jestWatch);
-registerTask('sass', sass);
-registerTask('ts:commonjs', ts.commonjs);
-registerTask('ts:esm', ts.esm);
-registerTask('ts:amd', ts.amd);
-registerTask('tslint', tslint);
-registerTask('ts:commonjs-only', ts.commonjsOnly);
-registerTask('webpack', webpack);
-registerTask('webpack-dev-server', webpackDevServer);
-registerTask('verify-api-extractor', verifyApiExtractor);
-registerTask('update-api-extractor', updateApiExtractor);
-registerTask('lint-imports', lintImports);
-registerTask('prettier', prettier);
-registerTask('bundle-size-collect', bundleSizeCollect);
-registerTask('check-for-modified-files', checkForModifiedFiles);
-registerTask('generate-version-files', generateVersionFiles);
-registerTask('perf-test', perfTest);
-
-task('ts', parallel('ts:commonjs', 'ts:esm', condition('ts:amd', () => argv().production && !argv().min)));
-
-task(
-  'build',
-  series(
-    'clean',
-    'copy',
-    'sass',
-    parallel(
-      condition('tslint', () => !argv().min),
-      condition('jest', () => !argv().min),
-      series(
-        argv().commonjs ? 'ts:commonjs-only' : 'ts',
-        condition('lint-imports', () => !argv().min),
-        // verify-api-extractor must always be run now because the api-docs package depends on its output
-        parallel(condition('webpack', () => !argv().min), 'verify-api-extractor')
+function preset() {
+  addResolvePath(__dirname);
```

Comment:
What does this do?

Reply:
this add s a resolve path for the build tooling deps like TS from the scripts folder